srcpkgs/kitty/template

```diff
@@ -36,7 +41,22 @@ do_install() {
 
 kitty-terminfo_package() {
 	short_desc+=" - terminfo data"
+	depends="ncurses"
```

Comment:
why is this needed? none of the other terminfo packages need it

Reply:
It was from this text in kitty's website:
> You need a couple of extra dependencies to build linux-package. tic to compile terminfo files, usually found in the development package of ncurses.

But yeah, if it compiles properly, then indeed there's no reason to have it there.

Reply:
tic compilation should be done at build-time